app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -137,6 +131,26 @@ export const Notifications = ({ setHeaderTitleNode }) => {
     );
   };
 
+  const onCopy = (notification) => {
+    trackEvent(SETTINGS_PAGE_EVENTS.CLONE_PATTERN_ICON);
```

Comment:
It is not pattern. It is a notification. Clarify with Web Analyst the corresponding event.

Reply:
It's my typo, sorry